File: src/lecture09/java/Main.java
package lecture09.java;

public class Main {

    public static void main(String[] args) {
        JavaPerson person = new JavaPerson("who-hoo", 100);
        System.out.println(person.getName());
        person.setAge(10);
        System.out.println(person.getAge());
    }
}

File: src/lecture09/kotlin/Main.kt
package lecture09.kotlin

import lecture09.java.JavaPerson

fun main() {
    val person = Person("who-hoo", 100)
    println(person.name)
    person.age = 10
    println(person.age)

    val baby = Person("baby")
    println(baby.name)
    println(baby.age)

    val hgd = Person()
    println(hgd.name)
    println(hgd.age)

    val javaPerson = JavaPerson("who-hoo", 100)
    println(javaPerson.name)
    javaPerson.age = 10
    println(javaPerson.age)
}
